Classes/JRHostsFile.h

```diff
@@ -1,5 +1,11 @@
 @interface JRHostsFile : NSObject
 
 -(id)init;
+-(id)initWithFileName:(NSString*)fileName;
+
+-(NSString*)defaultFileName;
+-(NSString*)getFileName;
+
+@property (nonatomic,strong) NSString* fileName;
```

Comment:
I don't think it makes sense to allow changing the file name after an instance of `JRHostsFile` has been created, so this should be read-only.